test/stream.spec.js

```diff
@@ -320,13 +311,17 @@ describe('stream', () => {
       await pipe(
         input,
         tap(msg => generatedMsgs.push(msg)),
-        tap(() => { if (i++ >= maxMsgs) initiator.abort(error) }),
         initiator,
         tap(msg => {
           if (msg.data) {
             msg.data = bufferToMessage(msg.data)
           }
+
           msgs.push(msg)
+
+          if (i++ >= maxMsgs) {
```

Comment:
This was a bit weird - `iso-random-stream` behaves a bit differently to `random-bytes` and was causing the initiator to abort immediately before the final message was echoed back from the receiver.  I think `random-bytes` causes execution to pause until the next tick of the event loop instead or something along those lines, so I moved the abort to after the echoed message collection.